ch data acquisition - restructure section: handle data securely (#451)
worldbank/d4di: chapters/data-collection.tex

and you should never decrypt it until it is downloaded on your computer.
If your data collection service allows you to browse data in the browser, 
then the encryption is only implemented correctly if you are asked for the key each time.

Finally, the data security standards that apply when receiving confidential data from the field
also apply when transferring confidential data to the field.
In some survey software, 
you can use the same encryption that allows you to receive data securely
from the field, to also send confidential data,
such as an identifying list of respondents, to the field.

\subsection{Receiving or sharing data securely}

If you receive confidential data from a partner organization 
or share confidential data with your research team over the internet,
it must always be encrypted.  
The best practice is to reduce the amount of confidential data
needed to be shared or transfer.
If you do not need the confidential information you will simplify your workflow
by collecting or receiving data without any confidential information in the first place.
This should always be your first option when possible.

> Usually we need to deal with confidential data at some point in the project, 

Reviewer: For the final review, we need to make sure we use the terms 'confidential' and 'personally-identifying" consistently throughout all sections/chapters

Author: Ok, opened up an issue for that here: https://github.com/worldbank/d4di/issues/454

and then the second best practice is create two versions of your data. 
One version of your data will still have the confidential information,
and it will need to be encrypted at all times. 
This version should only be used when the confidential information is actually needed.
In the second version of your data all confidential data should be removed.
This version does not have to be encrypted 
and should be what you use on a day-to-day basis.

The most common reason a data is confidential is that it includes personal identifiers. 
Removing identifying information is called de-identification,
and we will discuss how to do that in the next chapter. 

> De-identifying the data, at the earliest possible opportunity 

Reviewer: ```suggestion
De-identifying the data at the earliest possible opportunity 
```

will simplify the workflow without compromising data security,

> as de-identified data no longer must be encrypted.

Reviewer: ```suggestion
as de-identified data no longer needs to be encrypted.
```

> Another method of making data less confidential is to aggregate the data, 

Reviewer: ```suggestion
Another method to protect confidential data is to aggregate the data, 
```

meaning that sums and averages of group of observations are calculated
and only those numbers are reported.
Any of these techniques can be used either before the data is shared with the research team,
or when creating the second non-confidential version of the data set to be used in the analysis.

> If confidential information is directly required for the analysis itself,

Reviewer: ```suggestion
If confidential information is directly required for the analysis,
```

it will be necessary to keep at least a subset of the data encrypted through the data analysis process.

> There are multiple options on how to share data encrypted. 

Reviewer: ```suggestion
There are multiple options for sharing encrypted data. 
```

If you need to collaborate on a confidential data set 

> then you can create an encrypted folders using VeraCrypt as described above.

Reviewer: ```suggestion
then you can create an encrypted folder, for example using VeraCrypt (as described above).
```

An encrypted folder can be shared the same way you share a normal file
and it can even be shared using insecure modes of communication

> such as email or third-party syncing services.

Reviewer: Not clear to me how you would normally share files outside of email / sync software; suggest simplifying the wording as follows: 
An encrypted folder can safely be shared using insecure modes of communication such as email or third-party syncing services.

Author: implemented as suggested in e79e29f9ac376bfea501f95976ce45aaafddf1e6

Anyone who has access to both the encrypted folder and the encryption key
can read and collaborate on the encrypted file.

If you only need a one-time transfer of a confidential dataset, 
for example when receiving data from a partner organization,

> then the setting up of an encrypted folder

Reviewer: ```suggestion
then setting up an encrypted folder
```

> might not be your best option or even feasible.

Reviewer: why wouldn't it be feasible?

Author: It might be a bit too techy for some counterparts, and I did not know how to express that without either:
1. Make it sound as if encryption and VeraCrypt 
1. Make it sound as if we think that partner organization cannot be techy.

I will go over the other comments and come back to this to think of a better way to say it

Author: Made a new try based on a suggestion from Ben in, 9135f9b874c191761db5e2038252ac55504a5395. This says the same thing I want to say, without implying anything about partner organizations.

> Instead, for one-time transfers you can send data over password protected

Reviewer: ```suggestion
Instead, for one-time transfers we recommend sending data over password-protected
```

	You should not use the same encrypted folder or the same key as above, 
	because if you use the same key and lose the key, 
	then you will have lost access to both encrypted folders. 
	If you have an physical safe where you can securely store the external drive, 
	then you do not need to encrypt the data
	and thereby do not risk losing access by losing an encryption key.
	
	\item Finally, create a third encrypted folder.
	Either you can create this on your computer and upload it to a long-term cloud storage service,
	or you can create it on	an external hard drive that you then store in a second location,
	for example, at another office of your organization.
	This is the ``golden master'' backup copy of the raw data.
	You should never store the ``golden master'' copy in a synced folder, 
	as it would be deleted in the cloud storage if it is deleted on your computer.
	You should also never work with this data on a day-to-day basis.
\end{enumerate}

\noindent This handling satisfies the \textbf{3-2-1 rule}:
there are two on-site copies of the data and one off-site copy,
so the data can never be lost in case of hardware failure.\sidenote{

> 	\url{https://www.backblaze.com/blog/the-3-2-1-backup-strategy}}

Reviewer: I had removed that url because it is already cited in ch2

Author: Might have used outdated text here, as I do did not intentionally add it back. Anyways, removed in 775e293fb88cff27dfbbc11fe97d8184b7ef4f29
